### PR TITLE
Patch 6 updates capabilities

### PR DESCRIPTION
**Pull request by AndrewKeig**

### CODE REVIEW

File: UAT/V1.10.6_patch.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	cqc."ServicesCapacity"
+SET
+	"Type"='Utilisation'
+WHERE
+	"ServiceID" in (11, 13, 18, 20, 22, 35);
+
+
+INSERT INTO cqc."ServicesCapacity"
+("ServiceCapacityID", "ServiceID", "Question", "Sequence", "Type")
+VALUES(17, 19, 'Total places Number of people using the service on the completion date', 1, 'Capacity');
```
Comment (wozzer72) [inline UAT/V1.10.6_patch.sql]: How can this be of `Type` "capacity", if the scope of the card is to make all single questioned capacities of `Type` "Utilisation"?

File: UAT/V1.10.6_patch.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+UPDATE 
+	cqc."ServicesCapacity"
+SET
+	"Type"='Utilisation'
+WHERE
+	"ServiceID" in (11, 13, 18, 20, 22, 35);
```
Comment (wozzer72) [inline UAT/V1.10.6_patch.sql]: technically correct yes. 20, 13, and 11 are already `utilisations`. :-)

File: UAT/V1.10.6_patch.sql (hunk 1 above)
Comment (AndrewKeig) [reply UAT/V1.10.6_patch.sql]: fixed.

File: UAT/V1.10.6_patch.sql (hunk 2 above)
Comment (AndrewKeig) [reply UAT/V1.10.6_patch.sql]: fixed.

**Comment by wozzer72** [comment]
[(1) Capacities and Utilisation](https://trello.com/c/lyFtxmAc/81-1-capacities-and-utilisation)

**Pull request merged**